plugins/inputs/eventhub_consumer/README.md

```diff
@@ -88,11 +95,70 @@ The main focus for development of this plugin is Azure IoT hub:
   ## Each data format has its own unique set of configuration options, read
   ## more about them here:
   ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md
-  data_format = "influx"
```

Comment:
default should probably be left at influx.

Reply:
Thank you for a quick and thorough review!

No problem, I can change it back, but are you absolutely sure about it?  influx format is not what you see every day in kafka topics, json from the other side is one of the most popular options. The most popular should probably be default.

That line was the main reason of spending 2h investigating what's going on when I started using the plugin. The same line is the main reason #8643 was opened, because if the format is not correct, the message silently disappears.

Please, consider and let me know.

Reply:
Nvm, I moved back to influx, json docs have enough examples and there is a clear link to other data formats above this line.
To address issues like #8643 I will open another PR to improve logging when received data doesn't correspond to the specified format.